### ISSUE TITLE
[bug] Epics module not appearing on project admin page

### ISSUE THREAD

**Issue by kwill** [taigaio/taiga-back#971]
**Do you want to request a *feature* or report a *bug*?**
Bug.

**What is the current behavior?**
"Epics" module does not appear on admin page.

**If the current behavior is a bug, please provide the steps to reproduce.**

* Log in
* Visit project
* Visit admin page (cog icon)
* Visit modules page (see screenshot below)

**What is the expected behavior?**

* "Epics" module is listed and can be enabled

**Is it happening ~~in taiga.io or~~ in your own instance?**

* "Epics" module is not listed (see screenshot below)

This is not the same as https://github.com/taigaio/taiga-back/issues/834 because the issue persists after logging-out-then-in-again and after restarting the browser.

![taiga-screenshot](https://cloud.githubusercontent.com/assets/229415/24506209/0aff66ca-155d-11e7-8ee3-5736fa19232f.png)

**What browser/version are you using?**

* Firefox 52
* Taiga version unknown (see https://github.com/taigaio/taiga-front/issues/1276)

**Are there any console errors *(Ctrl + F12)* in red?**
N/A

**Comment by Xaviju** [comment]
Please add the version number from https://github.com/taigaio/taiga-front/issues/1276

**Comment by kwill** [comment]
```
root@taiga:/home/taiga/taiga-back# git describe --tag
2.1.0-33-g0f7e25b

root@taiga:/home/taiga/taiga-front-dist# git describe --tag
2.1.0-stable-1-g7bb8edd
```

[Epics were only released in 3.0.0](https://blog.taiga.io/taiga-stellaria-borealis-release-300.html) so... mystery solved :)
